tests/conftest.py

```diff
@@ -19,14 +20,15 @@ def get_spacy_model(
     ner: bool,
     with_custom_tokenizer: bool = False,
     with_sentence_segmenter: bool = False,
+    with_serializable_abbreviation_detector: bool = False,
 ) -> SpacyModelType:
     """
     In order to avoid loading spacy models repeatedly,
     we'll save references to them, keyed by the options
     we used to create the spacy model, so any particular
     configuration only gets loaded once.
     """
-    options = (spacy_model_name, pos_tags, parse, ner, with_custom_tokenizer, with_sentence_segmenter)
+    options = (spacy_model_name, pos_tags, parse, ner, with_custom_tokenizer, with_sentence_segmenter, with_serializable_abbreviation_detector)
```

Comment:
I don't think the `with_serializable_abbreviation_detector` param does exactly what it should. Setting it to `False` doesn't add the abbreviation detector at all, whereas I think setting it to False should add the non serializable version of the abbreviation detector. Perhaps two params `with_abbreviation_detector` and `serializable_abbreviation_detector`